pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "dictionaryutils"
+version = "3.0.3"
+description = "Python wrapper and metaschema for datadictionary."
+authors = ["CTDS UChicago <cdis@uchicago.edu>"]
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+
+[tool.poetry.dev-dependencies]
```

Comment:
should some of these be in `dependencies` instead of `dev-dependencies`? cdisutils, cdislogging, gen3datamodel

Reply:
yeah, so everything in the `setup.py` should be in `dependencies`. Do you know if PyPi only includes the dependencies listed in `setup.py` and not in `requirements.txt` ?

Reply:
i think pypi will only include setup.py dependencies